code/game/machinery/bots/medbot.dm

```diff
@@ -305,6 +305,17 @@ var/list/firstaid_exceptions = list(
 		on = 1
 		icon_state = "[icon_initial][on]"
 
+/obj/machinery/bot/medbot/emag_ai(mob/living/silicon/ai/A)
```

Comment:
same, handle this all at the bot level, for all these bots, I won't note on all of them